robocop/checkers/errors.py

```diff
@@ -205,6 +205,21 @@
         severity=RuleSeverity.ERROR,
         version=">=4.0",
     ),
+    "0414": Rule(
+        rule_id="0414",
+        name="duplicated-setting",
+        msg="{{ error_msg }}",
+        severity=RuleSeverity.ERROR,
```

Comment:
Do we want this rule to be an error or warning? Does this violation break the execution or just makes some code being ignored?

Reply:
Right, it would be better if it were warning. Also it should belong to duplications group. However since it's catched parsing error in that case we should catch it there (and ignore) and catch it again in duplications - oor I could try if the new design of rules allow to defining rules in one place and importing elsewhere (here). I will test it out

Reply:
Stwitched to warning but I had to move ``duplicated-setting`` to duplications while also catching (and ignoring) this error in errors.py. Importing the rule from other module is not that easy and it wasn't the good design to define the rule in one place and only use in the other.